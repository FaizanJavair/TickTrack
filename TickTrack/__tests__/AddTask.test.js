import React from "react";
import renderer from "react-test-renderer";

import AddTask from "../modals/addTaskModal";
import dataTemp from "../data";

it("Add Task renders correctly", () => {
  const tree = renderer.create(<AddTask list={dataTemp} />).toJSON();
  expect(tree).toMatchSnapshot();
});
