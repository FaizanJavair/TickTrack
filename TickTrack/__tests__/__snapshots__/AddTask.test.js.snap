// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Task renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "left": "7%",
        "opacity": 1,
        "position": "absolute",
        "top": "7%",
      }
    }
  >
    <Text />
  </View>
  <View
    onLayout={[Function]}
    style={
      [
        [
          undefined,
        ],
        {
          "paddingBottom": 0,
        },
      ]
    }
  >
    <Text
      style={
        {
          "color": "black",
          "fontSize": 30,
          "fontWeight": "600",
          "marginBottom": "5%",
          "textAlign": "center",
        }
      }
    >
      Create Your Task
    </Text>
    <TextInput
      onChangeText={[Function]}
      style={
        [
          {
            "borderRadius": 12,
            "borderWidth": 2,
            "height": 40,
            "paddingHorizontal": 10,
            "width": 330,
          },
          {
            "borderColor": undefined,
          },
        ]
      }
      value=""
    />
    <View>
      <View
        style={
          {
            "flexDirection": "row",
            "marginTop": "3%",
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "backgroundColor": undefined,
              "borderRadius": 10,
              "fontSize": 15,
              "opacity": 1,
              "padding": 8,
            }
          }
        >
          <Text
            style={
              {
                "color": "white",
                "fontSize": 16,
              }
            }
          >
            Due Date & Time
          </Text>
        </View>
        <Text
          style={
            {
              "fontSize": 18,
              "marginLeft": "2%",
              "marginTop": "2%",
            }
          }
        />
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "paddingVertical": "5%",
        }
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": undefined,
            "borderRadius": 12,
            "opacity": 1,
            "padding": 12,
            "width": "50%",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "white",
                "fontSize": 15,
                "fontWeight": "600",
                "textAlign": "center",
              },
            ]
          }
        >
          Add Task
        </Text>
      </View>
    </View>
  </View>
</View>
`;
